### DISCUSSION TITLE
First Result - PE535

### DISCUSSION THREAD

**Discussion by pennro-se** [inkstitch/inkstitch#1400] [Show and tell]
Thought I'd share my first result making something with InkStitch and my Brother PE535:
![result](https://user-images.githubusercontent.com/471907/138036202-e3aeaac7-9a9f-400c-96d1-c11d90c0cda4.png)
Made with misc. polyester thread on cotton muslin.

Can confirm that the Brother PE535 works great with exports from InkStitch (I'm not sure if it was the PES or PEC file as I put both on my USB). The result is a bit different than the visualizer, especially on the edge, but I can work around that.

My biggest issue was I managed to snap my first needle on the first layer, I was using tension level 4 but switched to level 3 and did the rest without another break. Though it could have been from me being too lazy to bring out my ironing board and add stabilizer. 😅

I also had some issues with the eyeshine (white), the needle was pulling the fabric so I ended up turning the machine manually for them to make sure the fabric was settled between stitches, made me wish it was a crank instead of a knob! I am guessing it was just design error, stacking too many stitches.

I'll attach my zip of exports in case anyone wants to compare (the colours won't match because I didn't plan ahead):
[Zip File](https://github.com/inkstitch/inkstitch/files/7378635/pattern.zip)

All in all, I think it was a great first experience. I'll definitely be using what I learnt for future projects.

**Comment by pennro-se** [comment]
Second project!
![20211021_185643](https://user-images.githubusercontent.com/471907/138381492-59608fed-0d57-44b4-bcf5-ee27d404cddb.jpg)

For this one I did the initial design with [Figma][figma.com] and then transferred it over to Inkscape to do final tweaks (e.g. the satin columns, fill directions) before exporting the design files.

My biggest hurdle for this one was just trying to keep the design aligned while I cut around the initial running stitch. You can see some of the fabric sticking out around the outside, I hadn't cut close enough because I was worried about cutting through my single layer of the stabilizer which unfortunately happened anyways causing me to have to patch more pieces of stabilizer during the project. Will use more layers from the start in the future.

Here's the zip of my exports for this project:
[Zip file](https://github.com/inkstitch/inkstitch/files/7393879/pattern.zip)


All in all, I think this one was a success given it was my second project on the machine. I'll just keep learning more til my results are near perfect!

**Reply by lexelby** [reply]
Nice work!  Patch borders are hard, definitely.
